GPflow/gplvm.py

```diff
@@ -111,12 +116,26 @@ def __init__(self, X_mean, X_var, Y, kern, M, Z=None, X_prior_mean=None, X_prior
     def build_likelihood(self):
         """
         Construct a tensorflow function to compute the bound on the marginal
-        likelihood.
+        likelihood for the training data (all dimensions).
+        """
+        return self._build_likelihood_graph(self.X_mean, self.X_var, self.Y, self.X_prior_mean, self.X_prior_var)
+
+    def _build_likelihood_graph(self, X_mean, X_var, Y, X_prior_mean=None, X_prior_var=None):
```

Comment:
please add a docstring to explain why this function is split out from the main `build_likelihood`. I guess it's because it gets reused at predict time?